### PR TITLE
FE-378: fix fields containing spaces during index creation

### PR DESCRIPTION
**Pull request by palashkaria**
### Scope & Purpose

- [x] :hankey: Bugfix

### Checklist

- [x] Tests
  - [x] Manually tested
- [x] :book: CHANGELOG entry made

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-378

### CODE REVIEW

**Review by shd8 — APPROVED** [review]
LGTM!

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* FE-378: fix fields containing spaces during index creation on web UI.
```
Comment (nikhil-varma) [inline CHANGELOG:4]: This is only to trim spaces in the start and end?

**Review by nikhil-varma — APPROVED** [review]
Approving with a confirmation question 🙂

File: CHANGELOG (hunk 1 above)
Comment (palashkaria) [reply CHANGELOG:4]: yeah, as we say enter comma separated field, we should trim that. I think spaces in field name itself could be valid on a db level

**Pull request merged**